lib/HeroBonus.h

```diff
@@ -174,8 +174,9 @@ class DLL_LINKAGE CAddInfo : public std::vector<si32>
 	BONUS_NAME(TWO_HEX_ATTACK_BREATH) /*eg. dragons*/	\
 	BONUS_NAME(SPELL_DAMAGE_REDUCTION) /*eg. golems; value - reduction in %, subtype - spell school; -1 - all, 0 - air, 1 - fire, 2 - water, 3 - earth*/ \
 	BONUS_NAME(NO_WALL_PENALTY)							\
-	BONUS_NAME(NON_LIVING) /*eg. gargoyle*/				\
-	BONUS_NAME(RANDOM_SPELLCASTER) /*eg. master genie, val - level*/ \
+	BONUS_NAME(NON_LIVING) /*eg. golems, cannot be rised or healed, only neutral morale */				\
+    BONUS_NAME(GARGOYLE) /* gargoyle is special than NON_LIVING, cannot be rised or healed */ \
+    BONUS_NAME(RANDOM_SPELLCASTER) /*eg. master genie, val - level*/ \

```

Comment:
@alexvins is my memory serves me well?
I guess bonuses need to be added to end of list to avoid save breakage.

Reply:
And of course it's need to be aligned with tabs not spaces.

Reply:
@ArseniyShestakov  You`re correct. Only at end of list